src/pico/pico-screens/screens/ssd1306_70_40.cpp

```diff
@@ -0,0 +1,117 @@
+#include "ssd1306_70_40.hpp"
+#include "pico/ssd1306.h"
+#include "shared.h"
+
+
+
+    // .spi      = PICO_DEFAULT_SPI_INSTANCE,
+    // .gpio_din = PICO_DEFAULT_SPI_TX_PIN,
+    // .gpio_clk = PICO_DEFAULT_SPI_SCK_PIN,
+    // .gpio_cs  = PICO_DEFAULT_SPI_CSN_PIN,
+    // .gpio_dc  = 20,
+    // .gpio_rst = 21,
+    // .gpio_bl  = 22,
+#define START_Y 24
+// +3 to center on 70 pixel width screen
+#define START_X (28 + 3)
+
+extern "C" {
+
+    static int16_t dithering_quant_error[DOWNSAMPLED_WIDTH] = {0};
```

Comment:
I tried to make dithering happen and it did not happen. Not sure what's wrong, it's my first time at the dithering rodeo.

While greyscale would benefit from the dithering that's an even bigger lift, and greyscale already looks 100x better than monochrome, so I'm probably going to leave it at that